3581/2/confirmation.rb

```diff
@@ -0,0 +1,34 @@
+require_relative './data_keeper.rb'
+
+module Confirmation
+  def selfie_attachment(*)
+    if selfie?
+      chat_session[session_key]['photo'] = payload['photo']
+
+      save_context :geolocation
+      respond_with :message, text: t(:geo)
+    else
+      save_context :selfie_attachment
+      respond_with :message, text: t(:not_selfie)
+    end
+  end
+
+  def geolocation(*)
+    if geo?
```

Comment:
I don't like short names. Some people can not get what is `geo` from the first time